src/libs/actions/Welcome/OnboardingFlow.ts

```diff
@@ -103,6 +115,11 @@ function startOnboardingFlow() {
 
 function getOnboardingInitialPath(): string {
     const state = getStateFromPath(onboardingInitialPath, linkingConfig.config);
+    const showBusinessModal = onboardingValues && 'signupQualifier' in onboardingValues && onboardingValues.signupQualifier === CONST.ONBOARDING_SIGNUP_QUALIFIERS.VSB;
```

Comment:
You've referenced the string `signupQualifier` a few times, should this also be in CONST?

Reply:
Lets make his a const